README.md

```diff
@@ -1,21 +1,40 @@
-# Devextreme UI Template Gallery
+# DevExtreme UI Template Gallery
 
-This repository includes responsive UI Templates for the most pupular UI/UX patterns in web LOB applications. Currently, this project is in the 'preview' stage and contains the following CRM and Project Management UI templates:
+This repository includes responsive UI Templates for the most popular UI/UX patterns in web LOB applications. These templates are available for Angular, React, and Vue.
 
-* [CRM. Contact List](https://devexpress.github.io/devextreme-ui-template-gallery/material.blue.light.compact/angular/default/#/crm-contact-list)
-* [CRM. Contact Details](https://devexpress.github.io/devextreme-ui-template-gallery/material.blue.light.compact/angular/default/#/crm-contact-details)
-* [Planning. Task List](https://devexpress.github.io/devextreme-ui-template-gallery/material.blue.light.compact/angular/default/#/planning-task-list)
-* [Planning. Task Details](https://devexpress.github.io/devextreme-ui-template-gallery/material.blue.light.compact/angular/default/#/planning-task-details)
+![DevExtreme UI Template Gallery](/images/ui-template-gallery.png)
+
+Take a look at the gallery **online** here:
+
+- [Angular](https://devexpress.github.io/devextreme-ui-template-gallery/material.blue.light.compact/angular/default/#/crm-contact-list)
+- [React](https://devexpress.github.io/devextreme-ui-template-gallery/material.blue.light.compact/react/default/#/crm-contact-list)
+- [Vue](https://devexpress.github.io/devextreme-ui-template-gallery/material.blue.light.compact/vue/default/#/crm-contact-list)
+
+Use the left menu to navigate through the CRM, Planning and Analytics views.
 
 The UI Templates have responsive layouts with DevExtreme UI components. Note that the components use sample data entities without any business logic. You can use these UI Templates in your project and adapt them to your specific business requirements.
 
-> The repository is in preview mode. Only Angular version is available for use. Vue and React projects are still under development.
+## Get Started
+
+To run these UI templates on your machine, clone this repository and follow the instructions below.
+
+1. Run the `npm i` command in the framework folder to install packages to the project and prepare the repository. 
```

Comment:
I think the first step should be
1. Choose a corresponding framework folder and go there
React - `cd packages\react`
Angular - `cd packages\angular`
Vue - `cd packages\vue`
2. Run the `npm i` command to install packages to the project.
...

On mac of course there are different slashes, we can give paths for both Windows and Mac but I think mac users know what to do :-)

Reply:
Noted